src/routes/login/+page.server.ts

```diff
@@ -1,11 +1,13 @@
 import { redirect } from "@sveltejs/kit";
-import { getOIDCAuthorizationUrl, getRedirectURI } from "$lib/server/auth";
+import { getOIDCAuthorizationUrl } from "$lib/server/auth";
+import { base } from "$app/paths";
 
 export const actions = {
-	default: async function ({ url, locals }) {
+	default: async function ({ url, locals, request }) {
 		// TODO: Handle errors if provider is not responding
+		const referer = request.headers.get("referer");
```

Comment:
Currently you get redirected to the frameless space instead of https://huggingface.co/spaces/. I suppose we would need another env var to solve that...? Not a big deal anyway.